datamol/io.py

```diff
@@ -151,6 +163,9 @@ def to_sdf(
     if isinstance(mols, pd.DataFrame):
         mols = dm.from_df(mols, smiles_column=smiles_column, mol_column=mol_column)
```

Comment:
on a side note here, doesn't seem that the mol is popped from the property dict of the dataframe when `mol_column`  is provided.

Reply:
Good catch. Will fix.

Reply:
Done and as a nice new feature, you also get a mol automatic detection :-)